### DISCUSSION TITLE
posts/word-zotero-citation/

### DISCUSSION THREAD

**Discussion by giscus** [leegical/leegical.github.io#5] [Announcements]
# posts/word-zotero-citation/

如何在 Word 中使用 Zotero 插入参考文献，并设定东北大学要求的参考文献引用列表格式。

https://haoyep.com/posts/word-zotero-citation/

**Comment by fqw1727252209** [comment]
棒

**Comment by surpoloyang** [comment]
好文！！

**Comment by surpoloyang** [comment]
博主，有个问题想请教一下，就是完成通过创建宏进行交叉引用后，鼠标在文中的脚注悬停按住ctrl+左键不可以跳转到参考文献对应处诶，只能先将脚注[*]选中后，才能通过按住ctrl+左键跳转到参考文献对应处，希望博主可以帮忙解答一下！谢谢！！

**Reply by leegical** [reply]
本质上是通过超链接来实现类似交叉引用的功能，所以用起来也和超链接一样。能用就行吧，暂时没有更好的办法 :joy:

补充：东北大学上交图书馆的论文PDF版本要求不能存在任何超链接。因此这个交叉引用没啥用，最后上交的时候还是要去掉

**Comment by lx0106** [comment]
好用好用，赞👍
